db/migrate/20161104185652_unique_change_note_for_content_item.rb

```diff
@@ -0,0 +1,52 @@
+class UniqueChangeNoteForContentItem < ActiveRecord::Migration[5.0]
```

Comment:
I have been toying with the idea of having a data_migration rake task namespace for things like this. WDYT?